django/contrib/admin/static/admin/css/dark_mode.css

```
@@ -31,10 +31,12 @@
       --close-button-bg: #333333;
       --close-button-hover-bg: #666666;
 
+      --blockquote-color: var(--body-medium-color);
+      --blockquote-border-color: var(--blockquote-color);
+
       color-scheme: dark;
     }
-  }
-
+}
```

```suggestion
  }

```
(aim to reduce the diff to only needed changes)